ttnn/cpp/ttnn/operations/conv/conv2d/device/multi_core_optimized_conv_sharded/optimized_conv_op_width_sharded_v2.cpp

```diff
@@ -572,7 +572,7 @@ operation::ProgramWithCallbacks multi_core_optimized_conv_width_sharded_v2_impl(
         compute_defines["SPLIT_READER"] = "1";
     }
 
-    if (false) {
+    if (packer_l1_acc) {
```

Comment:
pls add tests for l1 acc

Reply:
Done. I have enabled l1 acc in test_conv_ws